app/openai_whisper/core.py

```diff
@@ -48,7 +48,7 @@ def language_detection(audio):
     audio = whisper.pad_or_trim(audio)
 
     # make log-Mel spectrogram and move to the same device as the model
-    mel = whisper.log_mel_spectrogram(audio).to(model.device)
+    mel = whisper.log_mel_spectrogram(audio, n_mels=128).to(model.device)
```

Comment:
The value depends on the model. See https://github.com/openai/whisper/discussions/1778#discussioncomment-11008382. Use `model.dims.n_mels` instead.

Reply:
Thanks! I updated the PR